collector/benchmarks/README.md

```diff
@@ -64,6 +64,10 @@ These are artificial programs that stress one particular aspect of the
 compiler. Some are entirely artificial, and some are extracted from real
 programs.
 
+- **await-call-tree**: A tree of 9 async fns that await each other, creating a
+  large type composed of many repeated `impl Future` types. Such types caused
+  [poor performance](https://github.com/rust-lang/rust/issues/65147) in the
+  past.
```

Comment:
+1 for updating the docs!

Reply:
I just realized the `9` is no longer correct, removing it :)